diff --git a/Backends/Graphics4/OpenGL/Sources/kinc/backend/graphics4/OpenGL.c.h b/Backends/Graphics4/OpenGL/Sources/kinc/backend/graphics4/OpenGL.c.h
index 4c492e93..2b461dfc 100644
--- a/Backends/Graphics4/OpenGL/Sources/kinc/backend/graphics4/OpenGL.c.h
+++ b/Backends/Graphics4/OpenGL/Sources/kinc/backend/graphics4/OpenGL.c.h
@@ -724,7 +724,7 @@ void kinc_g4_scissor(int x, int y, int width, int height) {
 		glScissor(x, _renderTargetHeight - y - height, width, height);
 	}
 	else {
-		glScissor(x, y, width, height);
+		glScissor(x, _renderTargetHeight - y - height, width, height);
 	}
 }
 
